api/src/wfl/util.clj

```diff
@@ -401,8 +388,7 @@ vault.client.http/http-client                               ; Keep :clint eastwo
 (defn to-quoted-comma-separated-list
   "Return the sequence `xs` composed into a comma-separated list string.
   Example:
-    (to-quoted-comma-separated-list ['x 'y 'z])
```

Comment:
FYI the extra line is what you get when evaluating in a repl

Reply:
Didn't even realize I'd closed that.  More interested in the simpler quoting.